tensorflow/python/data/experimental/ops/readers.py

```diff
@@ -463,9 +468,11 @@ def make_csv_dataset_v2(
   Returns:
     A dataset, where each element is a (features, labels) tuple that corresponds
     to a batch of `batch_size` CSV rows. The features dictionary maps feature
-    column names to `Tensor`s containing the corresponding column data, and
-    labels is a `Tensor` containing the column data for the label column
-    specified by `label_name`.
+    column names to `Tensor`s containing the corresponding column data. If 
```

Comment:
Is the original doc accurate? IIUC there will only be a `labels` part of the tuple if `label_name` or `label_names` is set.

Perhaps we could say something like:

"A dataset, where each element is either a dictionary of features or, if `label_name` or `label_names` are specified, a (features, labels) tuple."

Reply:
There is a problem with the return format design. I think the general design is returning a ```(collections.OrderedDict() , collections.OrderedDict())``` tuple which is compatible with multiple label names. But it return a ```(collections.OrderedDict() , Tensor )``` now ...